src/frontend/README.md

```diff
@@ -35,4 +35,26 @@ stop: `nginx -s stop`
 
 ## start with nginx
 
-`npm run build` and start nginx from frontend directory with given conf file; see above. you should be able to access it on 0.0.0.0:8080
\ No newline at end of file
+`npm run build` and start nginx from frontend directory with given conf file; see above. you should be able to access it on 0.0.0.0:
+
+## REST API URL Configuration:
+
+### Dev Mode:
+ Set api base URL in `environments/environment.ts` file  
+
+### Production Mode:
+Set api base URL in `environments/environment.prod.ts` file  
+
+
+## Third Party Libraries used
```

Comment:
Used Libraries:

Make it a table

Library | Version |  Latest Stable (? Y/n) | License 
--- | --- | --- | --- |--- |

To Remove:

e.g.  Gauges?

Reply:
Also add, CSS and font assets  lib used?